### ISSUE TITLE
Referencing one manuscript from another

### ISSUE THREAD

**Issue by Ralph-Lee-UK** [BetaMasaheft/Documentation#1318]
I have two entries, EMIP 2091 and EMIP2092 that are catalogued as probably being part of the same codes. 

I am unclear as to exactly how to code this. Looking at the guidelines, found at https://betamasaheft.eu/Guidelines/?id=listRelation
I have come up with the following code, inserting @listRelation and then @relation commands in the history section, and in Atom this validates. Just wanting to clarify whether this is right, and in the correct place

<history>
                  <origin>
                     <origDate notBefore="1800" notAfter="1899">Nineteenth century</origDate>
                     <listRelation>
                       <relation name="dcterms:isPartOf" active="EMIP2091">
                         <desc>‎Tweed Codices 143 and 144 appear to be two fragments of the same ‎codex‎</desc>
                       </relation>
                     </listRelation>
                  </origin>
                  </history>

Does this seem reasonable? I should, of course, enter the same code in the other ms file too.

**Comment by Ralph-Lee-UK** [comment]
Pietro, my solution as detailed above seems not to have worked. I am sure there is a simple way to make such a reference to another record - can you advise?

**Comment by PietroLiuzzo** [comment]
Dear @Ralph-Lee-UK this is a yet underdocumented part of the guidelines, which has been waiting quite long now because of a publication timing. I have been promised this shortly for the last year, but as soon as it will appear I will include  more detailed this part and link it better with more examples. https://betamasaheft.eu/Guidelines/?q=syntaxe&id=syntaxe

To describe that there is an hypothetic manuscript which has no identification and may have been divided into two, which are those now identified, you would add this 

```xml
<listRelation>
<relation active="EMIP02091#UniCirc1" name="sdc:undergoesTransformation" passive="EMIP02091#tr1"/>
<!--here we declare an identifier, based on EMIP2091, but could be otherwise, for the original codex, and state it was transformed, we are also assigning an id to the transformation-->
<relation active="EMIP02091#UniCirc1" name="sdc:hasCertainty" passive="sdc:Hypothetique"/>
<!--we say with this statement that the above declared original codex is just an hypothesis-->
<relation active="EMIP02091#tr1" name="sdc:hasTransformationModel" passive="sdc:D3"/>
<!--D3 in la Syntax du codex is a simple division. we assign with this statement this model to the transformation declared above and we can procede with the following two statements in saying that this simple division produced our two manuscripts (which are by default UniCirc)-->
<relation active="EMIP02091#tr1" name="sdc:produces" passive="EMIP02091"/>
<relation active="EMIP02091#tr1" name="sdc:produces" passive="EMIP02092"/>
</listRelation>
```

you will see what this gets you in the /graph section, clicking on _Syntaxe_ at the top, once you have entered this.

**Comment by Ralph-Lee-UK** [comment]
I have tried this, and we will see with the pull request this week whether it works! One final question - since this involves two records, should a similar version of this, with the references reversed, be coded in the second record as well? I have coded this for EMIP 2093 (apologies I got the numbering wrong above) and I wonder whether it should also be added for EMIP 2092, or does the single statement sort out all the relations between the records?
